src/runtime/critical_cortexm.go

```diff
@@ -0,0 +1,13 @@
+// +build cortexm
+
+package runtime
+
+import "device/arm"
+
+func critical() uintptr {
+	return arm.DisableInterrupts()
+}
+
+func exitCritical(state uintptr) {
+	arm.EnableInterrupts(state)
+}
```

Comment:
While I think we need this functionality, I think it's best put in the runtime/interrupt package. What do you think? The package can provide a platform independent way to disable/restore interrupts (or provide critical sections, as they're often called). This can perhaps be done in a separate PR.

(I wanted to do that but never got around to actually doing it).

Reply:
Alright, I will move it there.